CMake/git.cmake

```diff
@@ -9,7 +9,7 @@ endfunction(get_git_tag)
 
 function(get_git_commit_hash output_var)

```

Comment:
Do you have an idea for a fall back on systems that do not have git, or the full repo? The build system on Launchpad appears to not have the git tree available at build time :(

Reply:
This PR adds the ability to pass the version like this:

```
-DVERSION_NUM=1.0.1 -DVERSION_SUFFIX="-myver"
```

I think that the numeric version should be in a file in the repo instead of a git tag to have a useful fallback.

Reply:
By the way, here is the OpenDingux Contrib package:
https://github.com/od-contrib/opks/blob/master/package/devilutionx/devilutionx.mk

OpenDingux Contrib is a new thing I just made 20 minutes ago :)

Reply:
@muzena do you think this could be made to work for the PPA buildes?

Reply:
I dont't know, how can I test this?

Reply:
If you can provide the version as a parameter for cmake like this: `cmake -DVERSION_NUM=1.0.1 ..` then it would solve the issue with the PPA missing a version number.

Reply:
Ok, this will work for stable release, it's not problem to change version in rules file for stable version when is released, but for git PPA -DVERSION_SUFFIX="-myver", it must be some way to auto add commit number.

Reply:
Ok, having it hard coded in a file has the obvious drawback that we might forget to update it, but it guess it's better then nothing and acceptable for now.